dags/eam_workday_xmatters_integration.py

```diff
@@ -13,12 +13,18 @@
 It creates a Jira ticket if the task fails.
 
 [docker-etl](https://github.com/mozilla/docker-etl/tree/main/jobs/eam-integrations)
+[telemetry-airflow](https://github.com/mozilla/telemetry-airflow/tree/main/dags/eam_workday_xmatters_integration.py)
 
 This DAG requires the creation of an Airflow Jira connection.
 
 #### Owner
 jmoscon@mozilla.com
 
+#### Tags
+* impact/tier_3
+* repo/telemetry-airflow
+* triage/record_only
+

```

Comment:
nit: are these notes typical/necessary given that they're defined below? i.e. two places to edit if this changes.